integration_test/Dialect/Handshake/buffer_init_none/buffer_init_none.mlir

```diff
@@ -1,16 +1,5 @@
 // REQUIRES: iverilog,cocotb
 
-// This test is executed with all different buffering strategies
-
-// RUN: hlstool %s --dynamic-firrtl --ir-input-level 1 --buffering-strategy=all --verilog --lowering-options=disallowLocalVariables > %t.sv && \
```

Comment:
HandshakeToHW does not support all buffering strategies, and that's why we no longer check this, right?

Reply:
Theoretically, it should support 'all', but not 'fifo'. I'll go ahead and add it in.

Reply:
`mix-std-handshake` and `tuple_packing` both fails due to https://github.com/llvm/circt/issues/5138 in the `all` strategy. For the sake of getting this in tree, i'll omit adding the `all` tests for those, for now...